drop-analytics/src/lib.rs

```diff
@@ -110,3 +123,30 @@ pub fn init_moose(
 pub fn moose_mock() -> Arc<dyn Moose> {
     Arc::new(mock_impl::MockImpl)
 }
+
+#[allow(unused_variables)]
+pub fn moose_file(
+    logger: Logger,
+    event_path: String,
+    app_version: String,
+    prod: bool,
+) -> Arc<dyn Moose> {
+    #[cfg(feature = "moose_file")]
```

Comment:
```suggestion
#[cfg(feature = "moose_file")]
pub fn moose_file(
    logger: Logger,
    event_path: String,
    app_version: String,
    prod: bool,
) -> Arc<dyn Moose> {
```

Reply:
This function is probably not even necessary, you can inline it since it consists of a single statement

Reply:
makes sense